packages/react/src/components/Grid/Grid-story.js

```diff
@@ -1,175 +1,158 @@
-    'undefined',
-    'Grid.Col'
-  ),
-  max: selectWithUndef(
-    'max',
-    withBlank(Grid.getValidColWidths().lgPlus),
-    'undefined',
-    'Grid.Col'
-  ),
-  smOffset: select('smOffset', Grid.getValidColOffsets().sm, 0, 'Grid.Col'),
-  mdOffset: select('mdOffset', Grid.getValidColOffsets().md, 0, 'Grid.Col'),
-  lgOffset: select('lgOffset', Grid.getValidColOffsets().lgPlus, 0, 'Grid.Col'),
-  xlgOffset: select(
-    'xlgOffset',
-    Grid.getValidColOffsets().lgPlus,
-    0,
-    'Grid.Col'
-  ),
-  maxOffset: select(
-    'maxOffset',
-    Grid.getValidColOffsets().lgPlus,
-    0,
-    'Grid.Col'
-  ),
-});
+export const responsiveGrid = () => (
+  <Grid>
+    <Row>
+      <Column sm={1} md={4} lg={8}>
+        <DemoContent>sm: 1/4, md: 1/2, lg: 2/3</DemoContent>
+      </Column>
+      <Column sm={1} md={2} lg={2}>
+        <DemoContent>sm: 1/4, md: 1/4, lg: 1/6</DemoContent>
+      </Column>
+      <Column sm={1} md={1} lg={1}>
+        <DemoContent>sm: 1/4, md: 1/8, lg: 1/12</DemoContent>
+      </Column>
+      <Column sm={1} md={1} lg={1}>
+        <DemoContent>sm: 1/4, md: 1/8, lg: 1/12</DemoContent>
+      </Column>
+    </Row>
+  </Grid>
+);
 
-const colors = () => ({
-  cell: color('cell (blue 10)', blue[10], 'colors'),
-  margin: color('margin (blue 20)', blue[20], 'colors'),
-  dividers: color('dividers (blue 40)', blue[40], 'colors'),
-  padding: color('padding (blue 40)', blue[40], 'colors'),
-  bleed: color('bleed (gray 100)', gray[100], 'colors'),
-});
+export const offset = () => (
+  <Grid>
+    <Row>
+      <Column smOffset={3} sm={1}>
+        <DemoContent>Offset 3</DemoContent>
+      </Column>
+      <Column smOffset={2} sm={2}>
+        <DemoContent>Offset 2</DemoContent>
+      </Column>
+      <Column smOffset={1} sm={3}>
+        <DemoContent>Offset 1</DemoContent>
+      </Column>
+      <Column smOffset={0} sm={4}>
+        <DemoContent>Offset 0</DemoContent>
+      </Column>
+    </Row>
+  </Grid>
+);
 
-export const Default = () => (
-  <div style={{ backgroundColor: colors().bleed }}>
-    <Grid {...propsGrid()} style={{ backgroundColor: colors().padding }}>
-      {Array.from({ length: 4 }).map((_, i) => (
-        <Grid.Row key={i} {...propsGridRow()}>
-          {Array.from({ length: 12 }).map((_, j) => (
-            <Grid.Col
-              key={j}
-              {...propsGridCol()}
-              style={{
-                outline: '1px dashed ' + colors().dividers,
-                backgroundColor: colors().margin,
-              }}>
-              <div className="outside">
-                <div
-                  className="inside"
-                  style={{ backgroundColor: colors().cell }}>
-                  <code>
-                    Row {i + 1}
-                    <br />
-                    Col {j + 1}
-                  </code>
-                </div>
-              </div>
-            </Grid.Col>
-          ))}
-        </Grid.Row>
-      ))}
-    </Grid>
-  </div>
```

Comment:
I think this and the below story are both really cool and fun to play around with, but I trust your knowledge that they're probably superfluous! Perhaps it's better omitted like this and left as a learning exercise of Storybook Knobs for myself.